### DISCUSSION TITLE
SpeakGPT web search

### DISCUSSION THREAD

**Discussion by AndraxDev** [AndraxDev/speak-gpt#125] [Ideas]
This discussion is for Issue https://github.com/AndraxDev/speak-gpt/issues/120

**Comment by thiswillbeyourgithub** [comment]
Just a heads up: openrouter allows access to perplexity models that have online search capabilities (they call them online models) so in a way this feature is already present in SpeakGPT.

Perplexity in itself maintains a fully openai compatible api actually so if you added their endpoints as a preset you could even directly mention in the readme that SpeakGPT supports web search :)

**Reply by AndraxDev** [reply]
This is already done. Please go to the Explore tab, then select Perplexity and ask the current date. This is a proof.

**Comment by AndraxDev** [comment]
Closing and locking this discussion due to inactivity.
